src/lib/plugins/commands.js

```diff
@@ -335,6 +335,30 @@ module.exports.server = function (serv, { version }) {
     return serv.selector(type, data)
   }
 
+  serv.parseSelectorString = (str, pos, world, allowUser) => {
+    if (/^@/.test(str) && !(/^@([arpe])(?:\[([^\]]+)\])?$/.test(str))) throw new UserError('Unknown selector type')
+    if (!(/^@/.test(str))) { return { str: serv.getPlayer(str) } }
+    return serv.selectorString(str, pos, world, allowUser)
+  }
+  /**
+   * @description send to a user a prismarine-chat message
+   * @param {string} aboutUser user the message is about
```

Comment:
This should be the player uuid, not username

Reply:
No, it's correct, this was done on purpose.